interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -268,6 +272,16 @@ namespace cling {
       return const_reverse_nested_iterator(0);
     }
 
+    void addModules(clang::Module* M) {
```

Comment:
We need to clearly distinguish `llvm::Modules` (which are until now the only `Module`s that a `Transaction` deals with) from `clang::Module`s. `addModules()` does not make that sufficiently clear.